tools/azure-rest-api-specs-examples-automation/directory/test_examples_dir.py

```diff
@@ -103,18 +103,45 @@ def test_find_resource_manager_example_typespec(self):
 
     def test_find_resource_manager_example_typespec_no_tsp_location(self):
         mock_path = path.abspath(".")
-        with self.assertRaises(RuntimeError) as context:
+        with self.assertRaises(RuntimeError):
             example_dir = try_find_resource_manager_example(
                 path.join(mock_path, "azure-rest-api-specs"),
                 path.join(mock_path, "azure-sdk-for-java/sdk/mongocluster/azure-resourcemanager-mongocluster"),
                 "2024-07-01",
                 "MongoClusters_ListConnectionStrings.json",
             )
 
+    def test_find_resource_manager_example_typespec_windows_directory_separator(self):
+        with create_mock_test_folder() as tmp_dir_name:
+            # use windows directory separator
```

Comment:
where do we use that?

Reply:
saw one in dotnet
https://github.com/Azure/azure-sdk-for-net/blob/Azure.ResourceManager.ContainerOrchestratorRuntime_1.0.0-beta.1/sdk/containerorchestratorruntime/Azure.ResourceManager.ContainerOrchestratorRuntime/tsp-location.yaml